include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1137,6 +1137,13 @@ FUNCTION(GetErrorValue, swift_getErrorValue, C_CC,
          ARGS(ErrorPtrTy, Int8PtrPtrTy, OpenedErrorTriplePtrTy),
          ATTRS(NoUnwind))
 
+// void __tsan_write8(void *addr);
+// This is a Thread Sanitizer instrumentation entry point in compiler-rt.
+FUNCTION(TSanInoutAccess, __tsan_write8, C_CC,
+         RETURNS(VoidTy),
+         ARGS(Int8PtrTy),
+         ATTRS(NoUnwind))
```

Comment:
That is the long-term plan. There is not a dynamically-sized callback for callback, so we (Kuba) will have to add one to compiler-rt. That is lower priority since we can get most of the benefits of checking without it. Instead we'll use __tsan_write1 (rather than 8, as I have here) for all inout accesses since that will avoid false positives from not including the size at the cost of false negatives.